src/pause.js

```diff
@@ -0,0 +1,64 @@
+class Pause extends Phaser.Scene {
```

Comment:
In Phaser, you can draw scenes on top of other scenes: this means that if we use `this.scene.play('pause')` in other places of the code, we can have the Pause scene render on top of those scenes, and then `this.scene.stop()` should return to whatever scene has been paused from.